### ISSUE TITLE
Not working on v13?

### ISSUE THREAD

**Issue by Yajo** [Tardo/OdooTerminal#18]
It's failing when I install a new odoo database in v13.

With debug=assets:

```
TypeError: service is undefined
    _call_service http://localhost:13069/web/static/src/js/core/service_mixins.js:89
    trigger http://localhost:13069/web/static/src/js/core/mixins.js:224
    _trigger_up http://localhost:13069/web/static/src/js/core/mixins.js:339
    _trigger_up http://localhost:13069/web/static/src/js/core/mixins.js:341
    trigger_up http://localhost:13069/web/static/src/js/core/mixins.js:334
    call http://localhost:13069/web/static/src/js/core/service_mixins.js:116
    _busServ moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/compat/v12/common.js:12
    init moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/compat/v12/common.js:17
    OdooClass.extend/Class.include/</prototype[name]</< http://localhost:13069/web/static/src/js/core/class.js:123
    Class http://localhost:13069/web/static/src/js/core/class.js:107
    init moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/terminal.js:306
    OdooClass.extend/</prototype[name]</< http://localhost:13069/web/static/src/js/core/class.js:90
    init moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/funcs/backend.js:16
    OdooClass.extend/Class.include/</prototype[name]</< http://localhost:13069/web/static/src/js/core/class.js:123
    init moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/funcs/core.js:11
    OdooClass.extend/Class.include/</prototype[name]</< http://localhost:13069/web/static/src/js/core/class.js:123
    init moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/funcs/common.js:19
    OdooClass.extend/Class.include/</prototype[name]</< http://localhost:13069/web/static/src/js/core/class.js:123
    Class http://localhost:13069/web/static/src/js/core/class.js:107
    <anonymous> moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/loaders/backend.js:23
    jQuery 11
    <anonymous> moz-extension://b4528c19-ae3f-420c-8805-2d57a522dc7f/odoo/js/loaders/backend.js:20
    def http://localhost:13069/web/static/src/js/boot.js:260
    processJob http://localhost:13069/web/static/src/js/boot.js:258
    processJobs http://localhost:13069/web/static/src/js/boot.js:316
    def http://localhost:13069/web/static/src/js/boot.js:271
    promise callback*processJob/def< http://localhost:13069/web/static/src/js/boot.js:267
    processJob http://localhost:13069/web/static/src/js/boot.js:258
    processJobs http://localhost:13069/web/static/src/js/boot.js:316
    def http://localhost:13069/web/static/src/js/boot.js:271
    promise callback*processJob/def< http://localhost:13069/web/static/src/js/boot.js:267
    processJob http://localhost:13069/web/static/src/js/boot.js:258
    processJobs http://localhost:13069/web/static/src/js/boot.js:316
    def http://localhost:13069/web/static/src/js/boot.js:271
    promise callback*processJob/def< http://localhost:13069/web/static/src/js/boot.js:267
    processJob http://localhost:13069/web/static/src/js/boot.js:258
    processJobs http://localhost:13069/web/static/src/js/boot.js:316
    def http://localhost:13069/web/static/src/js/boot.js:271
    promise callback*processJob/def< http://localhost:13069/web/static/src/js/boot.js:267
    processJob http://localhost:13069/web/static/src/js/boot.js:258
    processJobs http://localhost:13069/web/static/src/js/boot.js:316
    define http://localhost:13069/web/static/src/js/boot.js:154
    <anonymous> http://localhost:13069/web?debug=assets#action=35&cids=1&menu_id=5&model=ir.module.module&view_type=kanban:384
backend.js:37:21
```

**Comment by Tardo** [comment]
Thanks for report yajo! This happens when 'bus' module is not installed. I will do a workaround to not break the script execution if this part fails.

**Comment by Tardo** [comment]
Fixed in 6.1.0: https://github.com/Tardo/OdooTerminal/commit/001f58bd90932d7e821d2341dd502efbbcce9426
